app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -61,7 +61,7 @@ class MessageActivity : AppCompatActivity() {
         mode.adapter = ArrayAdapter<String>(this, spinnerItem, Mode.values().map { getString(it.stringResId) })
 
         send.setOnClickListener {
-            val inputManager: InputMethodManager = this?.getSystemService(Context.INPUT_METHOD_SERVICE)
+            val inputManager: InputMethodManager = this.getSystemService(Context.INPUT_METHOD_SERVICE)
```

Comment:
Was this really needed?

Reply:
Yes, This was called with a safe call keyword. But no safe call was needed as this variable is not nullable